lib/user-methods.js

```diff
@@ -275,6 +275,32 @@ function _authenticate(userConstructor, server, json, callback) {
     }
 }
 
+function _updateAccount(userConstructor, server, json) {
+    const url = append_url(server, 'auth/password/updateAccount');
+    const options = {
+        method: 'POST',
+        body: JSON.stringify(json),
+        headers: postHeaders,
+        open_timeout: 5000
+    };
+
+    const promise = performFetch(url, options)
+        .then((response) => {
+            const contentType = response.headers.get('Content-Type');
+            if (contentType.indexOf('application/json') === -1) {
+                return response.text().then((body) => {
+                    throw new AuthError({
+                        title: `Could not update user account: Realm Object Server didn't respond with valid JSON`,
+                        body,
+                    });
+                });
+            } else if (!response.ok) {
```

Comment:
The `else` is redundant here as there is a `return` in the previous `if` statement.